src/Checks/PreflightCheck.php

```diff
@@ -123,7 +123,7 @@ protected function checkConfig(Result $result): Result
         $missingKeys = [];
 
         foreach ($this->requiredConfig as $configKey) {
-            if (! config()->has($configKey) || empty(config($configKey))) {
+            if (! config()->has($configKey) || ! is_bool(config($configKey)) && empty(config($configKey))) {
```

Comment:
I don't know this codebase, so I may be talking gibberish here. But wouldn't it be better if we assigned `config($configKey)` to a variable instead of calling the function 3 times?

Like:

```php
$configValue = config($configKey);
if (! is_bool($configValue) && ! $configValue)) {
    //...
}
```

Reply:
It's only called twice and it's specifically inside that loop, but it's a good point, thanks for the suggestion, just pushed the change.